components/messages-inbox.tsx

```
@@ -0,0 +1,540 @@
+      } else {
+        setShowConversations(true)
+      }
+    }
+
+    handleResize()
+    window.addEventListener("resize", handleResize)
+    return () => window.removeEventListener("resize", handleResize)
+  }, [activeConversation])
+
+  // Scroll to bottom of messages
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: messages, activeConversation

This dependency can be removed from the list.

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)